SimG4CMS/Calo/interface/HFFibreFiducial.h

```diff
@@ -10,8 +10,6 @@
 class HFFibreFiducial {
 
 public:
-  HFFibreFiducial() {}
-  virtual ~HFFibreFiducial() {}
```

Comment:
@civanch from my understanding, the removal of constructor and destructor is justified by the fact that they are totally dummy, there is no single private data member to initialize, and this class in in practice just a wrapper for a function acting on an input vector, am I correct? BTW, I notice that the function is full of hardcoded magic numbers...

Reply:
@fabiocos , you are right! Because the function is static, no class object is needed. This function was full of LogInfo/Verbatim checks, which may help developer of this class but polluting debug printout with many lines. This makes a problem for debug SIM and it is the main motivation for this PR. The review and clean-up of the class itself requires a deep understanding of HF geometry and readout. This cannot be done now but may be formulated as an issue.

Reply:
perhaps the class should instead be a function in a namespace?

Reply:
@davidlange6 I would say so, looking at how is it used in a limited number of places...